tljh-plasmabio/tljh_plasmabio/__init__.py

```diff
@@ -83,31 +89,35 @@ def tljh_custom_jupyterhub_config(c):
     c.JupyterHub.hub_ip = public_ips()[0]
     c.JupyterHub.cleanup_servers = False
     c.JupyterHub.authenticator_class = PAMAuthenticator
-    c.JupyterHub.spawner_class = DockerSpawner
+    c.JupyterHub.spawner_class = SystemUserSpawner
     c.JupyterHub.allow_named_servers = True
     c.JupyterHub.named_server_limit_per_user = 2
 
     # spawner
     # increase the timeout to be able to pull larger Docker images
-    c.DockerSpawner.start_timeout = 120
-    c.DockerSpawner.pull_policy = "Never"
-    c.DockerSpawner.name_template = "{prefix}-{username}-{servername}"
-    c.DockerSpawner.default_url = "/lab"
-    c.DockerSpawner.cmd = ["jupyterhub-singleuser"]
-    c.DockerSpawner.volumes = {
-        os.path.join(VOLUMES_PATH, "{username}"): "/home/jovyan/work",
-        SHARED_DATA_PATH: {"bind": "/home/jovyan/data", "mode": "ro"},
+    c.SystemUserSpawner.start_timeout = 120
+    c.SystemUserSpawner.pull_policy = "Never"
+    c.SystemUserSpawner.name_template = "{prefix}-{username}-{servername}"
+    c.SystemUserSpawner.default_url = "/lab"
+    # TODO: change back to jupyterhub-singleuser
+    c.SystemUserSpawner.cmd = ["/srv/conda/envs/notebook/bin/jupyterhub-singleuser"]
+    c.SystemUserSpawner.volumes = {
+        os.path.join(os.path.dirname(__file__), "entrypoint.sh"): "/usr/local/bin/repo2docker-entrypoint",
```

Comment:
The `entrypoint.sh` is mounted here instead of having it in the environment repositories as a `start` file, so it stays in one place.